opentreemap/treemap/instance.py

```diff
@@ -442,6 +442,10 @@ def static_page_names(self):
 
         return names
 
+    def update_geo_rev(self):
+        self.geo_rev += 1
+        self.save()
```

Comment:
Good idea, done. I actually think it would be fine if two servers got the same updated `geo_rev` back in a race situation, but the F expression is  still the right choice.